apps/discovery_api/lib/discovery_api/data/table_info_cache.ex

```diff
@@ -0,0 +1,28 @@
+defmodule DiscoveryApi.Data.TableInfoCache do
+  @moduledoc """
+  Simple module to cache systemName to dataset_id mapping
+  """
+  require Logger
+
+  def child_spec([]) do
+    Supervisor.child_spec({Cachex, cache_name()}, id: __MODULE__)
+
+  end
+
+  def put(data, id) do
+    Cachex.put(cache_name(), id, data)
+    data
+  end
+
+  def get(id) do
+    Cachex.get!(cache_name(), id)
+  end
+
+  def invalidate() do
+    Cachex.clear(cache_name())
+  end
+
+  defp cache_name() do
+    :table_info
+  end
+end
```

Comment:
Not having looked at the other cache(s), would it make sense to consolidate any of these cachex boilerplate into a module that can be extended?

Reply:
@mcsearchin I know that system_name_cache does something very similar to this.

Reply:
Good idea. We decided to punt on this though, maybe we should write a card for it?